### ISSUE TITLE
Entity looks different when spawned with CustomEntity#spawn from API in v4

### ISSUE THREAD

**Issue by PlayerSchark** [PluginBugs/Issues-ItemsAdder#3724]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.

### What happened?

If you summon an entity with `/iaentity summon` it has the correct design.
If you summon an entity with `CustomEntity#spawn` the Model fails to render some of its parts (see screenshot).

### Potential cause:
#3721

### Steps to reproduce the issue

1. Create an entity
2. Spawn an entity with command
3. Spawn an entity with API

### Server version

Paper version 1.20.6-138-master@906df69

### ItemsAdder Version

ItemsAdder version 4.0.1-alpha-test-3

### ProtocolLib Version

ProtocolLib version 5.3.0-SNAPSHOT-713

### LoneLibs Version

LoneLibs version 1.0.54

### Error (optional)

```shell
No stacktrace (Error field is currently required)
```

### Screenshots/Videos (you can drag and drop files or paste links)

left: `/iaentity summon test:cannon`
right: `CustomEntity spawn = CustomEntity.spawn("test:cannon", location);`
![image](https://github.com/PluginBugs/Issues-ItemsAdder/assets/37115511/477f464e-86d6-4fb8-a3e6-676176dc0189)

**Comment by PlayerSchark** [comment]
It seems that due to the api allows rotation of the entity. The parts of the model seem to break.

spawned with 0.5PI (90deg) rotation
![image](https://github.com/PluginBugs/Issues-ItemsAdder/assets/37115511/c7b71d21-ea38-441e-a92b-459f14981824)

**Comment by LoneDev6** [comment]
Is this happening also with bundle false?

**Comment by PlayerSchark** [comment]
> Is this happening also with bundle false?

Yes it does. But I figured out if you set every pivot point to 0 0 0 expect the ones needed for actual rotation you can avoid this issue.

**Comment by LoneDev6** [comment]
Please provide a model for me to drag and drop and test as I cannot reproduce the issue.
Thanks for the patience
